code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -2446,16 +2527,258 @@
+	color = "#cca01c" // rgb(204, 160, 28)
+	strength = 20
+
+	glass_name = "Vesper"
+	glass_desc = "A dry martini, ice cold and well shaken."
+
+/datum/reagent/ethanol/rotgut
+	name = "Rotgut Fever Dream"
+	id = "rotgut"
+	description = "A heinous combination of clashing flavors."
+	taste_description = "plague and coldsweats"
+	color = "#3a6617" // rgb(58, 102, 23)
+	strength = 10
+
+	glass_name = "Rotgut Fever Dream"
+	glass_desc = "Why are you doing this to yourself?"
 
+/datum/reagent/ethanol/voxdelight
+	name = "Vox's Delight"
+	id = "voxdelight"
+	description = "A dangerous combination of all things flammable. Why would you drink this?"
+	taste_description = "corrosive death"
+	color = "#7c003a" // rgb(124, 0, 58)
+	strength = 10
+
+	glass_name = "Vox's Delight"
+	glass_desc = "Not recommended if you enjoy having organs."
+
+/datum/reagent/drink/voxdelight/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
+	..()
+	if(alien == IS_DIONA)
+		return
+	if(alien == IS_VOX)
+		M.adjustToxLoss(-0.5 * removed)
+		return
+	M.adjustToxLoss(3 * removed)
+
+/datum/reagent/ethanol/screamingviking
+	name = "Screaming Viking"
+	id = "screamingviking"
+	description = "A boozy, citrus-packed brew."
+	taste_description = "the bartender's frustration"
+	color = "#c6c603" // rgb(198, 198, 3)
+	strength = 9
+
+	glass_name = "Screaming Viking"
+	glass_desc = "A boozy, citrus-packed brew."
+
+/datum/reagent/ethanol/robustin
+	name = "Robustin"
+	id = "robustin"
+	description = "A bootleg brew of all the worst things on station."
+	taste_description = "cough syrup and fire"
+	color = "#6b0145" // rgb(107, 1, 69)
+	strength = 10
+
+	glass_name = "Robustin"
+	glass_desc = "A bootleg brew of all the worst things on station."
+
+/datum/reagent/ethanol/virginsip
+	name = "Virgin Sip"
+	id = "virginsip"
+	description = "A perfect martini, watered down and ruined."
+	taste_description = "emasculation and failure"
+	color = "#2E6671" // rgb(46, 102, 113)
+	strength = 60
+
+	glass_name = "Virgin Sip"
+	glass_desc = "A perfect martini, watered down and ruined."
+
+/datum/reagent/ethanol/jellyshot
+	name = "Jelly Shot"
+	id = "jellyshot"
+	description = "A thick and vibrant alcoholic gel, perfect for the night life."
+	taste_description = "thick, alcoholic cherry gel"
+	color = "#e00b0b" // rgb(224, 11, 11)
+	strength = 10
+
+	glass_name = "Jelly Shot"
+	glass_desc = "A thick and vibrant alcoholic gel, perfect for the night life."
+
+/datum/reagent/ethanol/slimeshot
+	name = "Named Bullet"
+	id = "slimeshot"
+	description = "A thick and toxic slime jelly shot."
+	taste_description = "liquified organs"
+	color = "#6fa300" // rgb(111, 163, 0)
+	strength = 10
+
+	glass_name = "Named Bullet"
+	glass_desc = "A thick slime jelly shot. You can feel your death approaching."
+
+/datum/reagent/drink/slimeshot/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
+	..()
+	if(alien == IS_DIONA)
+		return
+	M.reagents.add_reagent("slimejelly", 2)
```

Comment:
I would rather this do ToxLoss than add slime jelly, or if not, add tiny amounts of slime jelly.

Reply:
I'll drop the slime jelly amount down to .25 of a unit. The reasoning being that it'll heal prommies, but be deadly toxic to everything else.